### PR TITLE
reenable price widgets w/ fixes

### PR DESCRIPTION
**Pull request by benisgold**
Fixes RNBW-2208, RNBW-2161, RNBW-2167, RNBW-2168, RNBW-2169

## What changed (plus any additional context for devs)
**RNBW-2208**
- Modified provisioning profiles of widget targets to fix bug that caused testflight users to get logged out of the Rainbow app upon installing a build that includes the widgets.

**RNBW-2161**
- Currency selection in widget configuration now defaults to "Default" instead of "Choose"
- When you open the currency list it provides a subtitle under "Default" that specifies what the "default" currency is (this is the currency in the user's settings in the app)
- One caveat of including this subtitle is that it causes the blue "selected" checkmark to disappear when the default currency changes, since the subtitle changes with it.

**Question**: Is it worth including this subtitle at the cost of having this weird "selected checkmark" behavior?

**RNBW-2167**
- Widget displays $0.50 instead of $0.5

**RNBW-2168**
- Widgets using default display currency immediately refresh when the user changes the currency setting in the app. Before, it would refresh automatically on the 10 min schedule.

**RNBW-2169**
- Price change percentage is now according to widget display currency, not just USD.

## PoW (screenshots / screen recordings)
**This build is already on testflight if you want to play around with it - 1.6.2 (24)**

RNBW-2208
https://streamable.com/0kuhy6

RNBW-2161, RNBW-2168, RNBW-2169
https://user-images.githubusercontent.com/15272675/148880715-12b01a0e-56a8-4349-8db2-71292127f33d.mp4

RNBW-2167
Screen Shot 2022-01-10 at 7 11 02 PM

## Dev checklist for QA: what to test
Basically just that what I stated in the descriptions above is true, and nothing breaks.

## Addition requirements
Need to add this line to whitelist.json in rainbow-scripts. This dependency allows us to force refresh the widgets from RN.
```
"/react-native-widgetkit/src/index.tsx": {
    "Widgetkit": true
  }
```

### CODE REVIEW

**Comment by skylarbarrera** [comment]
merged dev and will get a fresh TestFlight build going so we can get this tested quicker

**Review by skylarbarrera — APPROVED** [review]
now a swift guy but code looks fine 👌 will do some QA once I get a TestFlight build going

**Comment by skylarbarrera** [comment]
latest testflight build for QA is 1.6.5 (12)

**Review by jinchung — APPROVED** [review]
🍔

**Review by brunobar79 — APPROVED** [review]
